### PR TITLE
fix: SE device UI issues

### PR DESCRIPTION
**Pull request by forgotvas**
Related to iOS Buggy UI #511 including header view for iOS 15

Contains fixes for:

- Learn Tab > Upgrade to access more features button has icon missing
- Course Home > UI does not load properly. The course header is covering the content. 
- All Course > Buggy UI
- Course dashboard > UI does not load properly. The course header is covering the content. 
- Course Dashboard > Expired course’s image disappears after a blink

### CODE REVIEW

**Review by saeedbashir — APPROVED** [review]
LGTM 👍

**Comment by forgotvas** [comment]
fixed header for iOS 15

**Comment by forgotvas** [comment]
Added some fixes for Bug Bash, description updated

**Comment by saeedbashir** [comment]
* The Shift Due Date is slightly under the header, on all devices
* On rotation the UI disturbs and shows UI of two tabs (only on iPhone 7 iOS 15)

Simulator Screenshot - iPhone 7 - 2024-07-25 at 08 43 45

https://github.com/user-attachments/assets/823efb8e-c594-4183-a61e-cd15aba1ff93

**Comment by saeedbashir** [comment]
@forgotvas Most of the iPad issues are fixed in this PR, but the value prop view is not rendered properly.
Simulator Screenshot - iPad (10th generation) - 2024-07-25 at 11 59 57

**Comment by forgotvas** [comment]
Hi @saeedbashir, i have fixed iPad and fixed Due Date top padding, for rotation bug please add note to Bug Bash. Thank you

**Review by rnr — APPROVED** [review]

**Comment by forgotvas** [comment]
@saeedbashir fixed rotation bug here too

**Comment by saeedbashir** [comment]
@forgotvas  Scroll is disabled on Course outline (Home) tab.
https://github.com/user-attachments/assets/bebaa3da-230c-4338-b04e-86819bdccd27

**Comment by forgotvas** [comment]
> @forgotvas Scroll is disabled on Course outline (Home) tab. https://github.com/user-attachments/assets/bebaa3da-230c-4338-b04e-86819bdccd27

fixed

File: Course/Course/Presentation/Container/CourseContainerView.swift [hunk 1]
```diff
@@ -316,7 +316,7 @@ public struct CourseContainerView: View {
             }
         }
         .versionedTabStyle()
-        .introspect(.scrollView, on: .iOS(.v15, .v16, .v17), customize: { tabView in
+        .introspect(.scrollView, on: .iOS(.v16, .v17), customize: { tabView in
```
Comment (saeedbashir) [inline Course/Course/Presentation/Container/CourseContainerView.swift]: @forgotvas  What will happen when the new iOS version iOS 18, will be released, do we need to white list future updates or will it be handled automatically?

Comment (forgotvas) [reply Course/Course/Presentation/Container/CourseContainerView.swift]: we will need add new version or we will need to change standard introspect to `@_spi(Advanced) import SwiftUIIntrospect` and use it like `.introspect(.scrollView, on: .iOS(.v15...), customize: { scrollView in`

Comment (saeedbashir) [reply Course/Course/Presentation/Container/CourseContainerView.swift]: Can you update it so we don't have to worry about future updates, until unless there is an issue?

Comment (saeedbashir) [reply Course/Course/Presentation/Container/CourseContainerView.swift]: or if it's out of the scope of this PR, we can create a task for this and can handle it post-MVP.

Comment (forgotvas) [reply Course/Course/Presentation/Container/CourseContainerView.swift]: updated, also checked via search on project word `.introspect` and found that there no any other places where it uses list like `.v(16), .v(17)`. Other places use certain version or relative range.

File: Core/Core/Assets.xcassets/trophy.imageset/Contents.json [hunk 2]
```diff
@@ -0,0 +1,12 @@
+{
+  "images" : [
+    {
+      "filename" : "Regular-M.pdf",
```
Comment (saeedbashir) [inline Core/Core/Assets.xcassets/trophy.imageset/Contents.json]: Could you please change the file name to the asset name?

**Review by saeedbashir — APPROVED** [review]
LGTM 👍

File: Core/Core/Assets.xcassets/trophy.imageset/Contents.json (hunk 2 above)
Comment (forgotvas) [reply Core/Core/Assets.xcassets/trophy.imageset/Contents.json]: done

**Pull request merged**